README.md

```diff
@@ -98,17 +98,17 @@ True if the storage implements `._stat`.
 
 True if the storage has been fully opened.
 
-#### `storage.suspendable`
-
-True if the storage implements `._suspend`.
-
```

Comment:
This wasn't actually implemented and would, if implemented, always be `true` as `storage.suspend()` defaults to `storage._close()` which has a default implementation.